contrib/sarplus/python/setup.py

```diff
@@ -0,0 +1,51 @@
+import sysconfig
+
+from setuptools import setup
+from setuptools.extension import Extension
+
+
+class get_pybind_include(object):
+    def __init__(self, user=False):
+        self.user = user
+
+    def __str__(self):
+        import pybind11
+
+        return pybind11.get_include(self.user)
+
+
+setup(
+    name="pysarplus",
+    version="0.2.6",
+    description="SAR prediction for use with PySpark",
+    url="https://github.com/Microsoft/Recommenders/contrib/sarplus",
+    author="Markus Cozowicz",
+    author_email="marcozo@microsoft.com",
+    license="MIT",
+    classifiers=[
+        "Development Status :: 4 - Beta",
+        "License :: OSI Approved :: MIT License",
+        "Programming Language :: Python :: 3.4",

```

Comment:
we're testing 3.5-3.7?

Reply:
see azure-pipelines.yml (also changed from 3.4 to 3.5-3.7). I can reduce the number of python environments.